docs/policy/repo-layout-migration-2025.md

```diff
@@ -0,0 +1,45 @@
+OSG Software Repo Layout Migration
+==================================
+
+11 December 2024
+
+On Monday, January 6th, 2025, the OSG Software Team will be upgrading the server that hosts the OSG Software repos
+(<https://repo.osg-htc.org> a.k.a. <https://repo.opensciencegrid.org>),
+which will result in changes to the directory layout for the OSG 23 and OSG 24 YUM repos.
```

Comment:
```suggestion
which will result in changes to the directory layout for the OSG 23 and OSG 24 Yum repos.
```

RedHat is all over the place on capitalization here but we should try to stay on the same page. I think we capitalize it elsewhere.